rfcs/support_virtual_test_suites.md

```diff
@@ -0,0 +1,47 @@
+# RFC 109: support virtual test suites in wptrunner
+
+## Summary
+
+Support virtual test suites in wptrunner, so that we can run tests with
+additional browser flags to exercise different code paths.
+
+## Details
+
+Add an *optional* command line argument to pass in a json configuration file. The
+content of the configuration file is defined by each vendor, and defines a list of
+virtual test suites. Each virtual test suite has a prefix, and a list of bases
+and args. For example, you could define a (hypothetical) virtual test suite
+using the following json configuration:
+
+```json
+{
+  "prefix": "css-highlight",
+  "bases": ["css/css-pseudo", "css/css-highlight-api"],
+  "args": ["--enable-features=css-highlight"]
+}
+```
+
+This will create new "virtual" tests of the form
+`virtual/css-highlight/css/css-pseudo/...` and
+`virtual/css-highlight/css/css-highlight-api/...` which correspond to the files
```

Comment:
In general I'm worried about reserving the `virtual` directory; so far we're only reserving `tools` `resources` and `common` at the top level. If we take this approach I think it should be `_virtual`.